### ISSUE TITLE
NVDA version alpha-31424,637e607b: Unable to reactivate Wasapi after deactivating it

### ISSUE THREAD

**Issue by paulber19** [nvaccess/nvda#16301]
### Actual behavior:

- select "disabled" for "Use WASAPI for audio output "  combo box.
- record NVDA configuration and restart NVDA.
- select "Enabled" for "Use WASAPI for audio output "  combo box.
- Press "Enter".

An error occurs and the modification is not taken into account.
The NVDA log:
ERROR - unhandled exception (22:22:55.899) - MainThread (13684):
Traceback (most recent call last):
  File "gui\settingsDialogs.pyc", line 721, in onOk
  File "gui\settingsDialogs.pyc", line 716, in _doSave
  File "gui\settingsDialogs.pyc", line 708, in _saveAllPanels
  File "gui\settingsDialogs.pyc", line 2813, in onSave
  File "audio\soundSplit.pyc", line 160, in setSoundSplitState
  File "audio\soundSplit.pyc", line 115, in applyToAllAudioSessions
AttributeError: 'NoneType' object has no attribute 'RegisterSessionNotification'
	

### Expected behavior:

No error and back to enabled state.
#### NVDA installed/portable/running from source:
NVDA Portable
#### NVDA version:
NVDA version alpha-31424,637e607b: 
#### Windows version:
Windows 10
#### Name and version of other software in use when reproducing the issue:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Adriani90** [comment]
cc: @mltony
